External/FEXCore/Source/Interface/Core/Interpreter/InterpreterCore.cpp

```diff
@@ -3698,6 +3733,30 @@ void InterpreterCore::ExecuteCode(FEXCore::Core::InternalThreadState *Thread) {
             }
             break;
           }
+          case IR::OP_FLOAT_TOGPR_S: {
```

Comment:
C/++ defines the cast as a truncation, so these don't actually use the host fpu flags. Also, x86 saturates negative /and/ positive overflows to 0x8000_0000, while arm does 0x8000_0000 and 0x7FFF_FFFF, so we might want to have manual overflow checks here?

Reply:
I think until we get some unit tests for this. At least until then were aren't actually doing worse than before.
I'll create an issue about it.